src/SourceCode.Clay/Sha1.cs

```diff
@@ -132,65 +154,57 @@ public static Sha1 Hash(in string value)
         public static Sha1 Hash(in byte[] bytes)
         {
             if (bytes == null) return Zero;
```

Comment:
Again, exception not Zero.

Reply:
That is a design change, I will tackle it in a separate PR